submissions/AnnaStoyano/memory-pair-game/index.js

```diff
@@ -0,0 +1,106 @@
+class Game {
+    constructor(cards) {
+        this.container = document.querySelector('.container');
+        this.cards = cards;
+    }
+
+    render() {
+        this.cards.forEach(card => {
+            this.container.insertAdjacentElement('beforeend', card.render());
+        });
+    }
+
+    checkOpenCards() {
+        let game = this;
+        let activeCardsNames;
+        this.container.addEventListener('click', function (event) {
```

Comment:
please, pay attention to the adding your event listeners. on each game restart you call the `checkOpenCards` function which adds a new event listener to `this.container`.